### DISCUSSION TITLE
Request device Information

### DISCUSSION THREAD

**Discussion by FreakErn** [Koenkk/zigbee2mqtt#12215] [General]
Is there any way to ask z2m for the last known device information?

For example if i could send:

`$ mosquitto_pub -h mqtt.fritz.box -t "zigbee2mqtt/`[TempSensor](https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html#xiaomi-wsdcgq11lm)`" -m '{}'`

To retrieve the last known information from z2m.

**Comment by Eggn1n3** [comment]
What about setting it to retain the value?
![image](https://user-images.githubusercontent.com/10974459/164004243-58cc51f1-d389-4fbd-b80b-a53fe108004b.png)

**Comment by FreakErn** [comment]
I am mostly interested in the last_seen and the battery information. A lot of devices send back to z2m some kind of "hey, i am here" request i cannot see it in mqtt - or at least i haven't seen the Message.
